pkg/kubernetes/informers.go

```diff
@@ -99,6 +99,20 @@ func (im *InformerManager) AddPVListener(add func(obj interface{}), update func(
 	})
 }
 
+// AddNamespaceListener hooks up add, update, delete callbacks
+func (im *InformerManager) AddNamespaceListener(add func(obj interface{}), update func(oldObj, newObj interface{}), remove func(obj interface{})) {
+	if im.namespaceInformer == nil {
+		im.namespaceInformer = im.informerFactory.Core().V1().Namespaces().Informer()
```

Comment:
Have considered using `NewFilteredNamespaceInformer` with `TweakListOptionsFunc` to watch only on workload namespaces?